common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -1356,6 +1356,17 @@ private boolean isTooLate(
         !appShuffleMergePartitionsInfo.shuffleMergePartitions.containsKey(reduceId);
     }
 
+    /**
+     * If necessary, look for the ignored bock bytes and capture them in pushMergeMetrics.
+     */
+    private void lookForIgnoredBlockBytesIfNecessary() {
+      if (deferredBufs != null && !deferredBufs.isEmpty()) {
+        for (ByteBuffer buf : deferredBufs) {
+          mergeManager.pushMergeMetrics.ignoredBlockBytes.mark(buf.limit());
```

Comment:
I think we should use `buf.remaining` here

Reply:
Sounds good,  have updated the PR.